sparkbox/apprentice-at
# Refactor: Local Dev Enviroment Update

Updates local server from an `express.js` server to use [browser-sync](https://browsersync.io/). This allows hot reloading and synchronized browser testing.

- adds `browser-sync` package for hot reloading
- adds `npm-run-all` package. This allows us to run both `browser-sync` and `grunt dev` from a single `npm start` script
- adds `npm start` script to run `grunt dev` & `browser-sync` together
- removes `express.js` and server
- updates README

### Testing

- run `npm start`
    - ensure browser-sync and `grunt dev` are both running
    - the local dev server starts up
- make changes in the code and verify changes are automatically reflected in the browser
